epub34/authoring/index.html

```diff
@@ -3109,854 +3108,921 @@ <h3>Parsing URLs in the package document</h3>
 					document as <var>base</var>.</p>
 			</section>
 
-			<section id="sec-shared-attrs">
-				<h3>Shared attributes</h3>
-
-				<p>This section provides definitions for shared attributes (i.e., attributes allowed on two or more
-					elements).</p>
+			<section id="sec-vocab-assoc">
+				<h3>Vocabulary association mechanisms</h3>
 
-				<section id="attrdef-dir" data-epubcheck="true"
-					data-tests="https://w3c.github.io/epub-structural-tests/#05-package-document_package-document.feature_L24">
-					<h4>The <code>dir</code> attribute</h4>
+				<section id="vocab-assoc-intro">
+					<h3>Introduction</h3>
 
-					<div class="note">
-						<p>The <code>dir</code> attribute currently does not have sufficient <a
-								href="https://www.w3.org/policies/process/#adequate-implementation">implementation
-								experience</a>, as [=reading systems=] typically only support a single default
-							directionality for metadata display.</p>
-						<p>Although it lacks the necessary reading system support, it is integral to the content model
-							on which EPUB is built (i.e., for internationalization support in the [=package document=]).
-							Consequently, it is strongly encouraged to set the proper directionality of text values in
-							the package document to ensure proper rendering once this situation improves.</p>
-						<p>The lessening of requirements for this attribute was only done to account for the different
-							process under which EPUB was developed prior to being brought into W3C. New features
-							developed under W3C processes will not receive a similar exemption.</p>
-					</div>
+					<p>EPUB defines a method of referencing terms and properties defined in metadata and semantic
+						vocabularies using the <a href="#sec-property-datatype"><var>property</var> data type</a>. The
+							<code>property</code> and <code>rel</code> attributes use the data type, for example, to
+						define properties and relationships in the [=package document=].</p>
 
-					<p
-						data-tests="#pkg-dir_creator-rtl,#pkg-dir_rtl-root-ltr,#pkg-dir_rtl-root-unset,#pkg-dir_unset-root-rtl,#pkg-dir_unset-root-unset,#pkg-dir-auto_root-rtl"
-						>Specifies the <a data-cite="bidi#BD5">base direction</a> [[bidi]] of the textual content and
-						attribute values of the carrying element and its descendants.</p>
+					<p>A <var>property</var> value is like a CURIE [[rdfa-core]] — it represents a URL [[url]] in
+						compact form. The expression consists of a prefix and a reference, where the prefix — whether
+						literal or implied — is a shorthand mapping of a URL that typically resolves to a term
+						vocabulary. When a [=reading system=] converts the prefix to its URL representation and combines
+						with the reference, the resulting URL normally resolves to a fragment within that vocabulary
```

Comment:
```suggestion
						vocabulary. When a [=reading system=] converts the prefix to its URL representation and concatenates
						with the reference, the resulting URL normally resolves to a fragment within that vocabulary
```

I'm just worried that the reader would believe that we are talking about a combination in the URL sense. There is a difference: if my prefix is `http://a.b.c` and the referent is `D`, a URL join would yield `http://a.b.c/D` where a CURIE resolution would yield `http://a.b.cD`. Hence, the importance of using a `/`, `#`, or a `?` character...

Reply:
I’m having trouble understanding the Vocabulary association mechanisms section. 
Can we simplify the text? If I’m having trouble, others might also. Particularly if English is a second language. Something like this
>EPUB defines a formal method of referencing terms and properties defined in metadata and semantic vocabularies using the [property data type](https://w3c.github.io/epub-specs/epub34/authoring/#sec-property-datatype). The epub:type attribute uses this data type in [EPUB content documents](https://w3c.github.io/epub-specs/epub34/authoring/#dfn-epub-content-document) and [media overlay documents](https://w3c.github.io/epub-specs/epub34/authoring/#dfn-media-overlay-document) to add [structural semantics](https://w3c.github.io/epub-specs/epub34/authoring/#app-structural-semantics), for example, while the property and relattributes use the data type to define properties and relationships in the [package document](https://w3c.github.io/epub-specs/epub34/authoring/#dfn-package-document).

Could be written to include just one example and state that other uses are possible.
>Here are formal ways to reference external vocabularies used in metadata attributes. These are in addition to definitions for EPUB-specific attribute vocabularies. 

>The [property data type](https://w3c.github.io/epub-specs/epub34/authoring/#sec-property-datatype) references the metadata vocabulary in use. For example, you can reference Dublin Core metadata when identifying an author. You would use the property data type to point reading systems to that vocabulary. 

>The epub:type attribute can be used in [EPUB content documents](https://w3c.github.io/epub-specs/epub34/authoring/#dfn-epub-content-document) and [media overlay documents](https://w3c.github.io/epub-specs/epub34/authoring/#dfn-media-overlay-document) to add [structural semantics](https://w3c.github.io/epub-specs/epub34/authoring/#app-structural-semantics). The property and rel attributes use the data type to define properties and relationships in the [package document](https://w3c.github.io/epub-specs/epub34/authoring/#dfn-package-document).

Reply:
Do we still have a "formal" in there? I'm sure I wrote out one instance just because it was bugging me when I looked at an introduction. What would an informal method even be doing in a specification?

The most confusing thing about CURIEs is that they look a lot like xml namespace declarations, so people expect to use xmlns:abc="" to declare them. And most reading systems don't even bother expanding curies. They just string match against the reserved prefixes we've defined.

But the CURIE itself is a compact way of writing a URL. The prefix is usually a reference to a dereferenceable vocabulary, but could just be a unique identifier in the form of a URL so you can use your own custom terms. You have to expand that prefix in order to get the base URL and the value after is the specific property or term or semantic being used. Fun stuff... if you're a semantic web geek!

Let me see what I can do with the intro, though.

Reply:
>Do we still have a "formal" in there? 

About the word “formal”— does it have a specific meaning in this context? Is it synonymous with “official” or “defined”? Or like “hey, it's supposed to be this way”?

*edited to put in the quote mark

Reply:
> does it have a specific meaning in this context?

Nope. It's just excess verbiage. Everything we define in the standard is formal and there is no possibility of informal methods in the package document.